folly/lang/ToAscii.h

```diff
@@ -226,7 +212,7 @@ FOLLY_ALWAYS_INLINE size_t to_ascii_size_clzll(uint64_t v) {
   }
 
   //  log2 is approx log<2>(v)
-  size_t const vlog2 = 64 - static_cast<size_t>(to_ascii_port_clzll(v));
+  size_t const vlog2 = 64 - static_cast<size_t>(__builtin_clzll(v));
```

Comment:
Presumably need to add the `#include` above.

Reply:
oops. Should be good now :)